src/test/java/com/revature/services/ReviewServiceTest.java

```diff
@@ -185,6 +226,20 @@ void testFindById() {
 		verify(this.mockReviewRepo, times(1)).findById(id);
 	}
 
+	@Test
+	void testFindById_Failure_ReviewIdNotFound() {
+		int id = 2;
+		given(this.mockReviewRepo.findById(id)).willReturn(Optional.empty());
+
+		try {
+			this.rServ.findById(id);
+			fail("Expected an exception to be thrown");
```

Comment:
You should specify which exception type is expected to be thrown. In this case, it's `ReviewNotFoundException`